### PR TITLE
APPS-2386 Add and "as of <date>" indicator to open/close text

### PR DESCRIPTION
**Pull request by bergieboy**

### CODE REVIEW

File: src/containers/location/providers/ProviderHeaderContainer.js [hunk 1]
```diff
@@ -114,10 +116,12 @@ const Header = styled.div`
 
 const SubHeader = styled.div`
   color: ${NEUTRAL.N700};
-  font-family: Inter;
+  display: flex;
+  flex-direction: row;
```
Comment (anzioka) [inline src/containers/location/providers/ProviderHeaderContainer.js:120]: you don't need this since it's default

**Review by anzioka — APPROVED** [review]

**Review by solomontang — DISMISSED** [review]

File: src/utils/DateUtils.js [hunk 2]
```diff
@@ -34,6 +34,16 @@ const isNowValid = (start :string, end :string) => {
...
 
+export const getLastUpdatedDate = (date :string) => {
+  const lastUpdatedDateTime = DateTime.fromISO(date);
+  const today = DateTime.local();
+  const yesterday = DateTime.local().minus({ days: 1 });
+  let dateLabel = getDateShortFromIsoDate(date);
+  if (lastUpdatedDateTime.hasSame(today, 'day')) dateLabel = 'Today';
+  if (lastUpdatedDateTime.hasSame(yesterday, 'day')) dateLabel = 'Yesterday';
+  return dateLabel;
+};
```
Comment (UnsungHero97) [inline src/utils/DateUtils.js:45]: Would love to see some version of this in `lattice-utils`.

File: src/containers/location/providers/ProviderHeaderContainer.js [hunk 3]
```diff
@@ -175,7 +186,12 @@ class ProviderHeaderContainer extends React.Component<Props> {
           <SubHeader>{`${city}, CA`}</SubHeader>
           <SubHeader>{type}</SubHeader>
           <SubHeader>{ages}</SubHeader>
-
+          <SubHeader>
+            <OpenClosedTag color={statusColor}>
+              {renderText(statusLabel)}
+            </OpenClosedTag>
+            <span>{`${renderText(LABELS.LAST_UPDATED)} ${lastUpdatedLabel}`}</span>
```
Comment (UnsungHero97) [inline src/containers/location/providers/ProviderHeaderContainer.js:193]: No change required. Let's start using the `Typography` component.

File: src/components/layout/index.js [hunk 4]
```diff
@@ -79,3 +79,11 @@ export const HeaderActions = styled.div`
...
   margin-left: auto;
 `;
+
+export const OpenClosedTag = styled.div`
+  color: ${(props) => props.color};
+  font-size: 14px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 17px;
+`;
```
Comment (UnsungHero97) [inline src/components/layout/index.js:89]: No change required. Let's start using the `Typography` component, which would replace `OpenClosedTag`.

**Review by UnsungHero97 — APPROVED** [review]

**Review by anzioka — DISMISSED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Review by solomontang — DISMISSED** [review]

**Review by UnsungHero97 — DISMISSED** [review]

**Review by solomontang — DISMISSED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Review by solomontang — APPROVED** [review]

**Pull request merged**